research/picai/nnunet_scripts/eval.py

```diff
@@ -30,18 +192,20 @@ def main() -> None:
     args = parser.parse_args()
 
     # Compute picai metrics
-    metrics = evaluate_folder(
+    # Warning, this method assumes binary segmentation (ie. only two classes)
```

Comment:
Curious why we aren't leveraging the get_picai_metrics here but we are in predict_and_eval.py